src/pages/experiments/[experimentId]/plots-and-tables/embedding-categorical/index.jsx

```diff
@@ -39,7 +39,7 @@ const route = {
 
 const EmbeddingCategoricalPlot = () => {
   const dispatch = useDispatch();
-  const config = useSelector((state) => state.plots[plotUuid]?.config);
+  const config = useSelector((state) => state.plots[plotUuid] ?.config);
```

Comment:
fix this

Reply:
I spent a long time trying to figure out why VScode adds a space on saving. Nothing worked so far fixing the problem.